Add supporter message feature

grokdesigns opened the pull request:
## Community Contribution License Agreement
By creating this pull request, I grant the project maintainers an unlimited,
perpetual license to use, modify, and redistribute these contributions under any terms they
choose, including both the AGPLv3 and the Fossorial Commercial license terms. I
represent that I have the right to grant this license for all contributed content.

## Description
Add supporter message from request https://github.com/orgs/fosrl/discussions/460

![Screen Recording 2025-04-07 at 14 48 02](https://github.com/user-attachments/assets/8eeb8f28-0066-4365-8214-8be89cec2750)
*Updated

## How to test?
Rebuild from source and update Docker image.

oschwartz10612 commented:
Thanks @grokdesigns we think this is really cool! We want to put this somewhere in the project but maybe not where you have it at the bottom of the page. The reason is we were trying to set up the supporter key in a way so that when you purchase a key you kind of remove the watermark and it looks stock so users are not bothered by it.

So I think we just should brainstorm where to put this. Where do you think is best that still shows that you have purchased a key but is out of the way? Maybe in the /admin portal or maybe in the drop down for the user in the top left only if they are in the admin group? Or maybe it can be turned on and off in the config? What do you think?

grokdesigns commented:
![pangolin_demo](https://github.com/user-attachments/assets/72fbedbf-e572-45fd-a474-784863fc08dd)
@oschwartz10612 @miloschwartz Thanks for the feedback. Totally understand wanting to keep it more subtle. As you can see in the GIF, the only indication they're a supporter when the user is in the interface is the added orange star next to the app name in the footer. On hover it has a popover thanking them and will still shoot some confetti when clicked. I also added a confetti/fireworks animation when a valid license key is entered. I changed all confetti colors to yellow/orange to match the theme a bit more.

Thoughts?

oschwartz10612 commented:
I think this looks awesome - exactly what we were after! Thank you so much for adding this!
